pkg/project/client.go

```diff
@@ -213,6 +215,9 @@ func getDesiredProjects(ctx context.Context, cfg *config.CLIConfig, opts Options
 	if strings.TrimSpace(p) == "" {
 		return nil, nil
 	}
+	if !opts.Create && len(projects) != 0 && !slices.Contains(projects, p) {
```

Comment:
nit: can this be simplified?
```suggestion
	if !opts.Create && !slices.Contains(projects, p) {
```

Reply:
yep had it in there before the opts.Create but no longer need now